dask_geomodeling/geometry/set_operations.py

```diff
@@ -77,11 +79,17 @@ def process(source_data, other_data=None):
 
 
 class Intersection(BaseSingle):
-    """Block that intersects geometries with the requested geometry.
+    """Calculate the geometric overlap/intersaction of two geometryBlocks.
```

Comment:
This is a weird one, and it is half implemented.

What it does now is something different than documented.
`Compute the geometric intersection of a GeometyBlock with the request geometry`

The second parameter is unimplemented and should be left out of the docs.